src/components/Cart.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import styled from "styled-components";
+import {Modal, Button} from "antd";
+
+const Context = styled.div`
```

Comment:
Context has a special meaning in React (https://reactjs.org/docs/hooks-reference.html#usecontext). Therefore I would name this styled component something like Container or Wrapper.